Source/Core/VideoCommon/FrameDump.cpp

```diff
@@ -69,9 +70,6 @@ void InitAVCodec()
   static bool first_run = true;
   if (first_run)
   {
-#if LIBAVCODEC_VERSION_MICRO >= 100 && LIBAVCODEC_VERSION_INT < AV_VERSION_INT(58, 9, 100)
-    av_register_all();
-#endif
```

Comment:
Why did you remove this? Do versions that are old enough to require this not compile anyway, making this check pointless?

Reply:
This `#if` was added to maintain compat with the version of ffmpeg in Externals before. It shouldn't be required anymore.

Reply:
Ah, okay. Probably fine to remove it then.